forge-gui/res/cardsfolder/upcoming/phoenix_chick.txt

```diff
@@ -0,0 +1,15 @@
+Name:Phoenix Chick
+ManaCost:R
+Types:Creature Phoenix
+PT:1/1
+K:Flying
+K:Haste
+K:CARDNAME can't block.
+T:Mode$ AttackersDeclared | CheckSVar$ X | SVarCompare$ GE3 | Execute$ TrigReturn | NoResolvingCheck$ True | TriggerZones$ Graveyard | AttackingPlayer$ You | TriggerDescription$ Whenever you attack with three or more creatures, you may pay {R}{R}. If you do, return Warcry Phoenix from your graveyard to the battlefield tapped and attacking with a +1/+1 counter on it.
+SVar:TrigReturn:AB$ ChangeZone | Cost$ R R | Defined$ Self | Origin$ Graveyard | Destination$ Battlefield | Tapped$ True | Attacking$ True | SubAbility$ AddCounter

```

Comment:
should be `WithCounterType$ P1P1`

Reply:
I'm a bit new to this, did a grep through the codebase and didn't find any references to "WithCounterType", and replacing the SubAbility portions with "WithCounterType$ P1P1" didn't seem to have the intended effect. Can you elaborate on what this means?

Reply:
ah typo its `WithCountersType$ P1P1`

Reply:
Should be good to go now.